### DISCUSSION TITLE
setup has a very execution, progress bar?

### DISCUSSION THREAD

**Discussion by Infinitay** [pycaret/pycaret#3255] [Q&A] [answered]
I'm trying to run PyCaret for time-series forecasting. My data has lots of missing timestamps, especially because my interval is 1-second, so I let pandas create the respect timestamp entries with `NaN`. I have 604K entries, but the filled data is around 19M entries. My PyCaret setup is the following, 

```py
s = setup(data=modified_df_ts.drop("id", axis=1), target="result", fh=60*60, use_gpu=True, numeric_imputation_target="ffill", numeric_imputation_exogenous="ffill", session_id=42, verbose=True)
```

As I said previously, it's taking very long. I've been running this for 14 hours now and it's still executing with no output in my notebook. I'm contemplating stopping the execution because I'm doing this on my local PC. That being said, is there a way I see anything to indicate the setup's progress to me? I assumed verbose would do just that, but I see nothing and it's been 14 hours.

**Comment by ngupta23** [comment, marked as answer]
Verbose will only output the summary results after setup is over. To see the progress in detail, you can check out the log file that is generated in the same folder and see the last step that was completed and where it is stuck.

**Reply by Infinitay** [reply]
Thank you. I can't believe I didn't notice the log file... that's embarrassing.
